tests/components/logbook/test_init.py

```diff
@@ -1339,36 +1536,36 @@ async def test_logbook_view_period_entity(hass, hass_client):
     # Test today entries without filters
     response = await client.get(f"/api/logbook/{start_date.isoformat()}")
     assert response.status == 200
-    json = await response.json()
-    assert len(json) == 2
-    assert json[0]["entity_id"] == entity_id_test
-    assert json[1]["entity_id"] == entity_id_second
+    response_json = await response.json()
+    assert len(response_json) == 2
+    assert response_json[0]["entity_id"] == entity_id_test
+    assert response_json[1]["entity_id"] == entity_id_second
```

Comment:
FYI - this change (and the subsequent ones in the logbook tests) was just to make pylint happy. The current code is using `json` as the variable name but it also has `import json` at the top so pylint was complaining about redefining variables. I just renamed them all to `response_json` so it would stop complaining, had nothing to do with glob filtering.

Reply:
Good call. I've had to fix that exact inside in other tests recently as well.